.gitignore

```diff
@@ -34,8 +34,3 @@
 # pytest #
 ##########
 *.pytest_cache/
-
-##########
```

Comment:
We totally want to ignore config.ini, that is the file that actually has my passwords and user-names in it. So we have to leave this in the .gitignore. I added that. Just remove the line *.csv.

Basically, the code uses config.ini, but we don't want the repo to track this file. So the repo tracks a file called example-config.ini that explains how to set up the real config file. This way, no sensitive information is shared. Hope this makes sense.